modules/desktop/graphics/ewwbar.nix

```diff
@@ -508,10 +505,10 @@ in
                 (sys_slider
                         :header "Volume"
                         :icon {volume.icon}
-                        :icon-onclick "${eww-volume}/bin/eww-volume mute &"
+                        :icon-onclick "PULSE_SERVER=audio-vm:4713 ${pkgs.pamixer}/bin/pamixer --toggle-mute &"
```

Comment:
I think these are duplicate code?
why not use eww-volume script here as before?

Reply:
Good point, we can sacrifice a few milliseconds of performance here since it's just a mute switch.